eq-author-api/config/knexfile.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     min: 10,
     max: 50
   },
+  acquireConnectionTimeout: 5000,
```

Comment:
Can we remove the waitforpostgres stuff with this? I think it’s a dev and ci only issue so it should be straightforward to test

Reply:
This is a connection timeout rather than a "retry for"